src/components/views/dialogs/QuestionDialog.tsx

```diff
@@ -39,7 +39,7 @@ export interface IQuestionDialogProps {
     onFinished(ok?: boolean): void;
 }
 
-export default class QuestionDialog extends React.Component<IQuestionDialogProps> {
+export default class ConfirmRedirectDialog extends React.Component<IQuestionDialogProps> {
```

Comment:
This doesn't seem right, should match the filename

Reply:
Yikes, sloppy!
Fixed